src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -76,6 +76,22 @@ def __init__(self, score_filepath: Path):
             "mixture temperature",
             "pressure",
             "environment temperature",
+            "annulus_a_temperature",
+            "annulus_b_temperature",
+            "annulus_c_temperature",
+            "annulus_d_temperature",
+            "annulus_e_temperature",
+            "annulus_a_pressure",
+            "annulus_b_pressure",
+            "annulus_c_pressure",
+            "annulus_d_pressure",
+            "annulus_e_pressure",
+            "wall_0_temperature",
```

Comment:
we should have a strategy to write only wall temperatures of walls that are really in the model. Otherwise it will write spurious (dummy) data to the output and can break SCORE. Maybe not a issue for now, but we should keep it in mind. Maybe annotate to review this in the future and create a task for this. Or you can think something now and implement here in this task.

Reply:
As we discussed, I added a conditional that ignores the wall temperature outputs if all values are negative, meaning they are dummies.

Reply:
Cool! Let's move then.